CMakeLists.txt

```diff
@@ -111,6 +111,8 @@ set(OT_CONFIG "" CACHE STRING "OpenThread config header file (deprecated, use `O
 
 set(OT_PROJECT_CONFIG "" CACHE STRING "OpenThread project-specific config header file")
 set(OT_PLATFORM_CONFIG "" CACHE STRING "OpenThread platform-specific config header file")
+set(OT_POSIX_FACTORY_CONFIG "" CACHE STRING "OpenThread factory config file")
+set(OT_POSIX_PRODUCT_CONFIG "" CACHE STRING "OpenThread product config file")
```

Comment:
Suggest moving these into the `sec/posix/CMakelist.txt` to make it clear it is applicable to POSIX platform?